docs/dxp/7.2.x/en/process-automation/README.md

```diff
@@ -35,15 +35,6 @@
 * APIs
   & ?
 
-## App Builder
```

Comment:
is this some kind of table of contents?

If so, I don't really get the difference between "Dev Guide" and "APIs". Aren't they supposed to talk about the same stuff? Or maybe put the APIs under "Dev Guide", if this is where we talk/show the available APIs.

Reply:
Yeah, I'd ignore this for now. It's probably going to change.